README.md

```diff
@@ -6,11 +6,16 @@ This is part of the [Mono project](http://www.mono-project.com/).
 
 This requires the libraries used by the Cairo vector graphics library to build (freetype2, fontconfig, Xft2 and libpng).
 
-On OSX you can use [Homebrew](https://brew.sh/) to install the dependencies:
+On **OSX** you can use [Homebrew](https://brew.sh/) to install the dependencies:
 
 	brew install glib cairo libexif libjpeg giflib libtiff autoconf libtool automake
 
-On Windows you can use [Chocolatey](https://chocolatey.org) to install the dependencies. Run the following commands from the root of the repository from an admin command prompt:
+On **Linux** you can use `apt-get` to install the dependencies:
+
+	sudo apt-get update
```

Comment:
I wouldn't mention this, we can assume the apt cache is up to date on a normal developer machine.

Reply:
Removed